content/it/DBaaS.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Database as a Service - DBaaS (Database come un Servizio)
+status: Completed
+category: Tecnologia
+---
+
+## Cos'è
+Database-as-a-Service (DBaaS) è ​​un servizio gestito da un operatore cloud (pubblico o privato) che supporta le applicazioni senza richiedere al team della app di eseguire le tradizionali funzioni di amministrazione del database. DBaaS consente agli sviluppatori di app di sfruttare i database senza essere esperti o senza assumere un amministratore di database per mantenere aggiornato il database.
+
+## Quale problema affronta
+Tradizionalmente, nelle configurazioni on-premise, le organizzazioni devono investire regolarmente in capacità di archiviazione e elaborazione aggiuntive per accogliere l'espansione del database che può essere costosa. Inoltre, gli sviluppatori effettuano il provisioning e configurano i database con l'aiuto dei team dell'infrastruttura IT, rallentando la velocità di distribuzione delle applicazioni basate su database. Anche il caricamento e l'esecuzione richiedono più tempo.
```

Comment:
cambierei giusto un paio di cose

- [ ] ... nelle installazioni on-premise 
- [ ] ... le organizzazioni devono regolarmente investire in ulteriore capacità di archiviazione ed elaborazione per consentire l'espansione del database, il che può risultare costoso
- [ ] ... gli sviluppatori effettuano il provisioning e configurano i database con l'aiuto dei team responsabili dell'infrastruttura IT e ciò rallenta la velocità di implementazione delle ...